system/Test/FeatureTestTrait.php

```diff
@@ -376,4 +390,41 @@ protected function populateGlobals(string $method, Request $request, array $para
 
 		return $request;
 	}
+
+	/**
+	 * Set the request's body formatted according to the value in $this->bodyFormat.
+	 * This allows the body to be formatted in a way that the controller is going to
+	 * expect as in the case of testing a JSON or XML API.
+	 *
+	 * @param  Request    $request
+	 * @param  null|array $params  The parameters to be formatted and put in the body. If this is empty, it will get the
+	 *                               what has been loaded into the request global of the request class.
+	 * @return mixed
```

Comment:
can use return Request: 

```suggestion
       * @return Request

Reply:
I made this change in my latest commit.